### DISCUSSION TITLE
Load balancing with particles

### DISCUSSION THREAD

**Discussion by arun-s-p** [AMReX-Codes/amrex#3369] [Q&A] [answered]
Hi, I am trying to invoke DistributionMapping with weights set to the number of particles per cell. However, I cannot find a way to extract the number of particles per cell which is to be assigned to the weights MultiFab. I can retrieve the number of particles in a tile using numParticles(). Is there an equivalent method for cells? 
Thanks

**Comment by atmyers** [comment, marked as answer]
You can use the `Increment` method of `ParticleContainer` to fill a Multifab with the number of particles in each cell, then include that in the weights.
